src/pageLayout/containers/MainNavigation.tsx

```diff
@@ -97,65 +286,67 @@ export const MainNavigation: FC = () => {
       userElement={CLOUD ? null : <UserWidget />}
       onToggleClick={handleToggleNavExpansion}
     >
-      {generateNavItems(org.id, operatorRole).map((item: NavItem) => {
-        const linkElement = (className: string): JSX.Element => (
-          <Link
-            to={item.link}
-            className={className}
-            title={item.label}
-            onClick={() => {
-              event('nav clicked', {which: item.id})
-            }}
-          />
-        )
-        return (
-          <TreeNav.Item
-            key={item.id}
-            id={item.id}
-            testID={item.testID}
-            icon={<Icon glyph={item.icon} />}
-            label={item.label}
-            shortLabel={item.shortLabel}
-            active={getNavItemActivation(
-              item.activeKeywords,
-              location.pathname
-            )}
-            linkElement={linkElement}
-          >
-            {Boolean(item.menu) && (
-              <TreeNav.SubMenu>
-                {item.menu.map((menuItem: NavSubItem) => {
-                  const linkElement = (className: string): JSX.Element => (
-                    <Link
-                      to={menuItem.link}
-                      className={className}
-                      onClick={() => {
-                        event('nav clicked', {
-                          which: `${item.id} - ${menuItem.id}`,
-                        })
-                      }}
-                    />
-                  )
+      {generateNavItems(org.id, operatorRole, shouldShowNotebooks).map(
```

Comment:
To see a more true diff, add `?w=1` or go to [this convenience url](https://github.com/influxdata/ui/pull/6422/files?w=1) to see the diff without whitespace changes.